docs/src/main/sphinx/sql/grant.rst

```diff
@@ -48,6 +48,19 @@ Grant ``SELECT`` privilege on the table ``orders`` to everyone::
 
     GRANT SELECT ON orders TO ROLE PUBLIC;
 
+Grant ``SELECT`` privilege on all tables in a schema to user ``bob`` with a wildcard::
```

Comment:
In both cases it might be more than just tables.. like views, materialized views and not sure what else..